src/tools/index.ts

```diff
@@ -1,5 +1,5 @@
-import {Tool} from 'sanity'
 import {SplitVerticalIcon} from '@sanity/icons'
+import {Tool} from 'sanity'
```

Comment:
Not sure if there is a guideline on this yet, but generally prefer a more explicit from of type imports like
```
import {type Tool} from 'sanity'
// or
import type {Tool} from 'sanity'
```

Reply:
There's an eslint rule that can enforce that. Remix ships with it.